src/NuGet.Clients/NuGet.PackageManagement.UI/Utility/UIUtility.cs

```diff
@@ -64,15 +64,17 @@ public static string NumberToString(long number, IFormatProvider culture)
 
             while (v >= 1000)
             {
-                v /= 1000;
+                v = Math.Round(v / 1000, 2);
```

Comment:
On PMUI we don't see 3 digit precision, most 2 digits like below, otherwise previous logic having too detailed info causes problem.
![image](https://user-images.githubusercontent.com/8766776/132720918-e638e2bd-2c52-450d-a4c5-a702aed52729.png)